dipy/nn/utils.py

```diff
@@ -126,4 +143,60 @@ def recover_img(image, affine, ori_shape, scale=2):
     """
     new_image, _ = reslice(image, np.eye(4), (scale, scale, scale), (1, 1, 1))
     recovered_img = affine_transform(new_image, affine, output_shape=ori_shape)
+    affine[:3, 3] += np.array([init_shape[0]//2,
+                               init_shape[1]//2,
+                               init_shape[2]//2])
+    if voxsize is not None and np.any(voxsize != np.ones(3)):
+        R = voxsize
+        kwargs = {'order': 1,
+                  'mode': 'constant',
+                  'matrix': R,
+                  'output_shape': image_shape}
+        recovered_img = np.round(affine_transform(recovered_img, **kwargs))
     return recovered_img
+
+
+def get_largest(binary_img):
+    r"""
+    Function to select only the largest background
```

Comment:
`function to` to remove. start with `Select`

also raw string does not seems to be needed, you can remove the `r"""`